src/main/java/uk/gov/pay/directdebit/webhook/gocardless/exception/InvalidWebhookExceptionMapper.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.pay.directdebit.webhook.gocardless.exception;
+
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class InvalidWebhookExceptionMapper implements ExceptionMapper<InvalidWebhookException> {
+
+    private static final Logger LOGGER = PayLoggerFactory.getLogger(InvalidWebhookExceptionMapper.class);
+
+    @Override
+    public Response toResponse(InvalidWebhookException exception) {
+        // Log an error and possibly do SumoLogic alert
+        LOGGER.error(exception.getMessage());
+
+        // Webhooks with an invalid signature must return a "498 Token Invalid" error.
+        // see https://developer.gocardless.com/api-reference/#webhooks-overview
```

Comment:
- +1 documented. 
- I like Java docs style /** and @see anotations, etc, consider to change (is integrated with ides)
- The exception is Gocardless specific so might be worth naming it accordingly (this is a minor since we know only Gocardless is used)

Reply:
Thanks, addressed. Just skipped the renaming of `InvalidWebhookExceptionMapper` to `GoCardless*` because as discussed I am using `uk.gov.pay.directdebit.webhook.gocardless.exception` package which is `GoCardless` specific

Reply:
👍